lisa/features/gpu.py

```diff
@@ -186,6 +189,19 @@ def _install_grid_driver(self, driver_url: str) -> None:
             "NVIDIA-Linux-x86_64-grid.run",
             executable=True,
         )
+        if isinstance(self._node.os, CBLMariner):
+            sed = self._node.tools[Sed]
+            sed.substitute(
+                match_lines="^GRUB_CMDLINE_LINUX",
+                regexp="lockdown=integrity",
+                replacement="lockdown=0",
+                file="/etc/default/grub",
+                sudo=True,
+            )
+            self._node.execute(
+                "grub2-mkconfig -o /boot/grub2/grub.cfg", shell=True, sudo=True
+            )
+            self._node.reboot()
```

Comment:
@squirrelsc  @LiliDeng 

Is this approach fine? The GRID driver is unsigned and is failing to install because lockdown=integrity is set in Mariner 2 by default

Reply:
It's ok to remove sign in test environments.

Reply:
I will Skip GRID Driver for Mariner, since it is not in the supported list.
https://learn.microsoft.com/en-us/azure/virtual-machines/linux/n-series-driver-setup#nvidia-grid-drivers